common/constants.gradle

```diff
@@ -5,6 +5,6 @@ project.ext.submodules = ['database', 'auth', 'storage']
 project.ext.group = 'com.firebaseui'
 project.ext.version = '1.0.1'
 project.ext.pomdesc = 'Firebase UI Android'
-project.ext.buildTools = '25.0.2'
```

Comment:
@CodingDoug the `version-1.1.0-dev` branch has the latest everything (build tools, support lib, gradle build plugin, etc...). Changing the base branch of this PR to `version-1.1.0-dev` would minimize merge conflicts when merging it back into `master`.